custodial-copy-re-indexer/src/test/scala/uk/gov/nationalarchives/reindexer/ReIndexerSpec.scala

```diff
@@ -0,0 +1,140 @@
+package uk.gov.nationalarchives.reindexer
+
+import cats.effect.unsafe.implicits.global
+import cats.effect.IO
+import fs2.Chunk
+import org.scalatest.EitherValues
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers.*
+import uk.gov.nationalarchives.reindexer.Arguments.ReIndexArgs
+import uk.gov.nationalarchives.reindexer.Configuration.{Config, FileType, IoUpdate}
+
+import java.io.ByteArrayInputStream
+import java.util.UUID
+import javax.xml.parsers.DocumentBuilderFactory
+import javax.xml.xpath.{XPathExpression, XPathFactory}
+
+class ReIndexerSpec extends AnyFlatSpec with EitherValues:
+  val uuids: List[UUID] = List(UUID.randomUUID, UUID.randomUUID, UUID.randomUUID)
+  val xpath: XPathExpression = XPathFactory.newInstance.newXPath.compile("//B")
+  val appConfig: Config = Config("", "", "")
```

Comment:
You have this here and within each test but are they being used?

Reply:
No, they're not, not sure why I did that. I've taken them out.